retrieve.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import json
+import re
+from bs4 import BeautifulSoup
+
+def userInputJson():
```

Comment:
i think we want to include like queries and location, but we can add that later, looks good for now :3